.github/actions/pr_diff/script.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+# Default base branch
+BASE_BRANCH="develop"
+
+# Target branch (to be provided as an argument)
+TARGET_BRANCH=""
+
+# GitHub repository owner
+OWNER="strapi"
+
+# GitHub repository name
+REPO="strapi"
+
+# Parse command-line arguments
+while getopts "b:t:o:r:" arg; do
```

Comment:
is it necessary considering it's always passed as env vars ?

Reply:
It was initially a script I wrote to be run manually, and then thought about adapting it for GitHub actions. I can remove it, but it seemed harmless to keep it here just in case.

I can remove it if you want?

Reply:
Yeah I don't know if it could have any side effect on how it's run in the CI so I would remove it. let's only keep code that runs

Reply:
I've removed the CLI arguments and only rely on env variables now